website/public/assets/css/_general.sass

```diff
@@ -92,17 +95,28 @@ body
       border-color: white
       border-width: 2px
       color: white
+      -webkit-font-smoothing: antialiased
+      -moz-osx-font-smoothing: antialiased
+
       ::-webkit-input-placeholder
```

Comment:
Remove all the pseudo element declarations. This can be covered by adding one instance to .form-control